themes/bootstrap5/js/bs3-compat.js

```diff
@@ -0,0 +1,150 @@
+/*global VuFind*/
+VuFind.register('bs3-compat', function cookie() {
```

Comment:
Could we consider a more descriptive name for this function?

Also, not a priority, but some of the below might be able to be moved into build.xml. Eventually.

Reply:
Oops, that was a silly one. I changed the module and function name to a more descriptive one.

Reply:
And yes, some of the changes could perhaps be moved to build.xml (or just done manually), but the intention here is to allow also bootstrap3-based themes with their own templates to work properly with minimal changes. In the long run I expect this to be changed to opt-in, or get deprecated and finally removed.